docs/Software Engineering Blog/posts/spring-boot-3-1-docker-compose-support-intuitive-consistent-develop-run-tests-containers.md

```diff
@@ -0,0 +1,239 @@
+---
+date:
+  created: 2024-09-01
+authors:
+  - NoahHsu
+categories:
+  - Spring Boot
+tags:
+- Java
+  - Development
+  - Unit Test
+  - Integration Test
+  - Docker
+---
+
+# Spring Boot 3.1 Docker Compose Support: The Most Intuitive and Consistent Way to Develop and Run Tests with Containers
+
+![SB-Docker-cover.png](resources%2Fsb-docker%2FSB-Docker-cover.png)
+
+[Spring-Boot-Docker-Compose](https://docs.spring.io/spring-boot/reference/features/dev-services.html#features.dev-services.docker-compose) is a very powerful tool to help developer in their daily job. Help the Spring Boot app start with a `docker-compose up` automatically. In Both development and running test. This tiny automation actually bring huge benefit to both developer experiment and engineering practice
```

Comment:
⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Weasel] 'tiny' is a weasel word!